docs/overview.md

```diff
@@ -48,40 +48,27 @@ technologies involved and be able to set up the basic stuff yourself.
-   ZFS is, it assumes the data in memory is correct, so [very bad
-   things](http://research.cs.wisc.edu/adsl/Publications/zfs-corruption-fast10.pdf)
-   happen to your data if there is memory corruption. For this reason, it is
-   [strongly
-   recommended](https://github.com/zfsonlinux/zfs/wiki/FAQ#do-i-have-to-use-ecc-memory-for-zfs)
-   to use ECC RAM. ZFS also prefers to have the hard drives all to itself. If
-   you're paranoid (a good mindset when dealing with servers), you'll probably
-   want an uninterruptible power supply (UPS) of some sort as well and SMART
-   monitoring for your hard drives. See the [FreeNAS hardware
-   requirements](https://freenas.org/hardware-requirements/) as a guideline.
+   RAM, and it is [recommended](zfs_overview.md) you use ECC RAM. ZFS also
+   prefers to have the hard drives all to itself. If you're paranoid (a good
+   mindset when dealing with servers), you'll probably want an uninterruptible
+   power supply (UPS) of some sort as well and SMART monitoring for your hard
+   drives. See the [FreeNAS hardware
+   requirements](https://freenas.org/hardware-requirements/) as a guideline, but
+   remember you'll also be running Docker.
 
 1. Install **Ubuntu Server**, preferably a Long Term Support (LTS) edition such
    as 18.04, and keep it updated. You'll probably want to perform other basic
    setup tasks like hardening SSH and including email notifications. There are
    [various guides](https://devanswers.co/ubuntu-18-04-initial-server-setup/)
    for this, but if you're just getting started, you'll probably need a book.
 
-1. Install **ZFS** and set up storage. This includes creating data sets for
-   various parts of the system, some form of automatic snapshot handling, and
-   possibly automatic backups to another server or an external hard drive.
-   Currently on Linux, it is [something of a
-   hassle](https://github.com/zfsonlinux/zfs/wiki/Ubuntu-18.04-Root-on-ZFS) to
-   use ZFS on the root file system. If you are completely new to ZFS, expect a
-   brutal learning curve. There is a slightly dated (2012) but extensive
-   [introduction to ZFS on
-   Linux](https://pthree.org/2012/04/17/install-zfs-on-debian-gnulinux/) by
-   Aaron Toponce to get you started, or you can watch [this
-   video](https://www.youtube.com/watch?v=MsY-BafQgj4) that introduces the
-   philosophy and big picture of ZFS.
-   
+1. Install **ZFS** and set up storage. You can use a different file system and
+   volume manager, but Ansible-NAS historically tends towards ZFS. You'll have
+   to create datasets for various parts of the system, some form of automatic
+   snapshot handling, and possibly automatic backups to another server or an
+   external hard drive. If you are completely new to ZFS, expect a brutal
+   learning curve. A [brief introduction](zfs_overview.md) is included here, as
+   well as a [basic example](zfs_configuration.md) of a very simple ZFS setup.
```

Comment:
So this is an interesting one and not something that occurred to me on first review. Few things that come to mind: 

- If someone has installed Ubuntu, then they have a root filesystem already, likely ext4, so their requirement is for an *additional* filesystem for storage, and "You'll have to create datasets for various parts of the system" is not strictly true. 
- ZFS packages are installed by Ansible-NAS for the purpose of migrating from FreeNAS and for using as the storage filesystem. There's no reason (that I can think of) where it would need to be installed manually.
- I'm not a fan of the negativity in "If you are completely new to ZFS, expect a brutal learning curve". I learnt the basics required in about 30 minutes, it ain't that hard, and I don't want people scared off before they even start :)

Reply:
I'm rewriting the overview text to leave the choice of "specialized bulk storage" more open, point out Btrfs (which at some point in the future is bound to get its RAID5 problems solved, and then should see an uppick in popularity) is an option, so we're more agnostic towards file systems. I cut out the ECC and VM parts here completely, because that can be taken care of in the ZFS texts themselves.